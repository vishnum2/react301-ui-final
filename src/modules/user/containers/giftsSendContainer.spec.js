import React from 'react';
import {shallow} from '../../../enzyme';
import {GiftsSendContainer} from './giftsSendContainer';

describe('snapshot', () => {
    it('should do snapshot test', () => {
        const wrapper = shallow(<GiftsSendContainer fetchSentCards={jest.fn()}  />);
        expect(wrapper).toMatchSnapshot();
        const error = new Error('test');
        wrapper.simulateError(error);
    });

    it('check and render login', () => {
        const wrapper = shallow(<GiftsSendContainer sentCards={[]} isLoggedIn={true} fetchSentCards={jest.fn()}  user={{user: 'new'}} />);
        expect(wrapper.find('h2')).toHaveLength(1);
    });
});
