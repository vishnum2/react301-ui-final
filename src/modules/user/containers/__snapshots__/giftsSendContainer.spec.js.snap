// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot should do snapshot test 1`] = `ShallowWrapper {}`;
